tree.scss

```diff
@@ -209,6 +209,7 @@ $timing-fn: ease-out;
       li.node {
         @include base-node;
         cursor: pointer;
+        overflow-y: hidden;
```

Comment:
I don't think it should. I don't think we have any cases where a node should have content going outside of it. This prevents a small scrollbar from being added to the node because the toggler is big enough now to reach into the top/bottom padding of the node-contents.